api-samples/devtools/chrome-query/chrome-query.md

```diff
@@ -0,0 +1,15 @@
+# devtools
```

Comment:
Should this file be named readme.md?

Reply:
Sorry, my mistake, the naming convention is README.md, can you please update the name agian.